Added Learn SQL in a Month of Lunches

stjepanjurekovic opened the pull request:
Hi,

Stjepan from Manning here. I thought this latest SQL title from our catalog might be a good match for your list. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added Learn SQL in a Month of Lunches

Hi,

Stjepan from Manning here. I thought this latest SQL title from our catalog might be a good match for your list. Thank you for considering it.

Best,

CLAassistant commented:
[![CLA assistant check](https://cla.pingcap.net/pull/badge/signed)](https://cla.pingcap.net/pingcap/awesome-database-learning?pullRequest=57) <br/>All committers have signed the CLA.

stjepanjurekovic committed:
Merge branch 'master' into master

stjepanjurekovic committed:
Added an article to Blogs section

Hi,

Stjepan from Manning here again. I have also added an article published in our Free Content Center last week. Thank you for considering it.

Best,

stjepanjurekovic commented:
Hi,

Sorry for the late reply. I have signed the agreement.

Best,


On Mon, Nov 21, 2022 at 10:30 AM CLAassistant ***@***.***>
wrote:

> [image: CLA assistant check]
> <https://cla-assistant.io/pingcap/awesome-database-learning?pullRequest=57>
> Thank you for your submission! We really appreciate it. Like many open
> source projects, we ask that you sign our Contributor License Agreement
> <https://cla-assistant.io/pingcap/awesome-database-learning?pullRequest=57>
> before we can accept your contribution.
> You have signed the CLA already but the status is still pending? Let us
> recheck
> <https://cla-assistant.io/check/pingcap/awesome-database-learning?pullRequest=57>
> it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pingcap/awesome-database-learning/pull/57#issuecomment-1321758483>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDIEMLNQSM6T3YVY2VTQKDWJM6LRANCNFSM6AAAAAASGNU2F4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Stjepan Jureković
Manning Publications - Marketing
***@***.***

stjepanjurekovic committed:
Added ScyllaDB in Action

Hi,

Stjepan here from Manning. I thought this title might be a good match for your list. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added DuckDB in Action

Hi,

Stjepan here from Manning. I thought this title might be a good match for your list. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added Just Use Postgres!

Hi,

Stjepan from Manning here. I thought this title might be a good match for your list. Thank you for considering it.

Best,

stjepanjurekovic committed:
Merge pull request #1 from stjepanjurekovic/stjepanjurekovic-patch-1

Added Just Use Postgres!
